src/decorator-annotator.ts

```diff
@@ -266,7 +267,7 @@ export class DecoratorClassVisitor {
           // rewriting. Note that we cannot use param.type as the emit node directly (not even just
           // for mapping), because that is marked as a type use of the node, not a value use, so it
           // doesn't get updated as an export.
-          const sym = this.typeChecker.getTypeAtLocation(param.type).getSymbol();
+          const sym = this.typeChecker.getTypeAtLocation(param.type).getSymbol()!;
```

Comment:
Correct.  We'll only have saved this param as a thing to investigate if it had a symbol.